src/diffusers/pipelines/animatediff/pipeline_animatediff_video2video.py

```diff
@@ -917,6 +886,14 @@ def __call__(
         latent_timestep = timesteps[:1].repeat(batch_size * num_videos_per_prompt)
 
         # 5. Prepare latent variables
+        # video must be a list of list of images
+        # the outer list denotes having multiple videos as input, whereas inner list means the frames of the video
+        # as a list of images
+        if video and not isinstance(video[0], list):
+            video = [video]
+        if latents is None:
```

Comment:
maybe raise an error in `check_inputs` when both `latents` and `video` is not `None`

Reply:
Already there:
https://github.com/huggingface/diffusers/blob/35358a2decad0c1e2972a8311ff49bfc14a326e1/src/diffusers/pipelines/animatediff/pipeline_animatediff_video2video.py#L598